File: pacman/apt-gui.pl
#!/bin/perl
use File::Basename;
use File::Spec;
require File::Spec->catfile(dirname(__FILE__), 'apt-base.pl');
our @pkgs;

push @pkgs, qw(
dbus-1-x11
geany
 pavucontrol
 pulseaudio
 pulseaudio-module-bluetooth
 thai-fonts                                 
 xdg-dbus-proxy
 lklug-fonts             
 indic-fonts         
 xdm
 xinit
 xhost
 
 xf86-input-evdev                      
 xf86-input-joystick                   
 xf86-input-keyboard                   
 xf86-input-libinput                   
 xf86-input-mouse                      
 xf86-input-synaptics                  
 xf86-input-vmmouse                    
 xf86-input-void                       
 xf86-input-wacom        
 
 xf86-video-amdgpu
 xf86-video-ark
 xf86-video-ati
 xf86-video-intel
 xf86-video-nouveau
xmessage
xorg-x11-server
xorg-x11-server-extra
yast2
yast2-qt
);

unless (caller){
    load(@ARGV);
}

File: pacman/apt-gui.py
#!/bin/python3
#use File::Basename;
#use File::Spec;
#require File::Spec->catfile(dirname(__FILE__), 'apt-base.pl');
#our @pkgs;
j=__import__('apt-base')
pkgs=j.pkgs

qemu=(
'libvirt',
'qemu',
'virt-manager',
"libvirt-daemon-driver-network",
"libvirt-daemon-driver-qemu",
'gnome-disk-utility',
'gparted',
'remmina-plugins-rdp',
'remmina-plugins-vnc',
'remmina'
)

if pkgs.check("LIVEINSTALL"):
    pkgs.extend((
"calamares",
))

if pkgs.check("RESCUEINSTALL"):
   pkgs.extend(qemu)
   qemu = []

if pkgs.check("EXTRAINSTALL"):
    pkgs.extend(qemu)
    pkgs.extend((
'nekoray',
'libreoffice-writer',
'libreoffice-calc',
'vscodium',
'wine',
'wine-mono',
'copr-cli',
'dnfdragora'
))

pkgs.extend((
#"dbus-x11",,
"xonsh",
"generic-logos",
"google-roboto-mono-fonts",
#"open-sans-fonts",
"openh264",
"pipewire",
"pipewire-pulseaudio",
"pipewire-media-session",
"pulseaudio-utils",
"xdg-dbus-proxy",
"xdg-utils",
#"xhost",
#"xinit",
#"xmessage",
#"xorg-x11-drv-intel",
#"xorg-x11-drv-nouveau",
#"xorg-x11-server-Xephyr",
#"xorg-x11-server-Xorg",
))

if __name__ == '__main__':
    pkgs.main()

